packages/ai-jsx/src/core/errors.ts

```diff
@@ -24,9 +24,12 @@ export enum ErrorCode {
   AnthropicDoesNotSupportSystemMessage = 1019,
   AnthropicDoesNotSupportFunctions = 1020,
   AnthropicAPIError = 1021,
+
   ChatModelDoesNotSupportFunctions = 1022,
   ChatCompletionBadInput = 1023,
 
+  Llama2DoesNotSupportFunctions = 1024,
```

Comment:
what's the difference between this code and 1022? As the number of supported models grows, seems like having per-model codes is going to get unwieldy.

Reply:
hehehe we had this same convo the last time I added a model provider: https://github.com/fixie-ai/ai-jsx/pull/165#discussion_r1245929073.

I hear your feedback and will continue to watch for the pain points noted in that original comment.